app/controllers/observations_controller.rb

```diff
@@ -5,23 +5,23 @@ class ObservationsController < ApplicationController
   rescue_from Pundit::NotAuthorizedError, with: :auth_errors
 
 
-  def index
-    @observer = User.new
-    @observation = Observation.new(user: @observer, proposal: @proposal)
-  end
-
   def create
-    observation = @proposal.add_observer(params[:observation][:user][:email_address])
+    cleaned = params.permit(observation: { user: [:email_address] })
+    email = cleaned.require(:observation).require(:user).require(:email_address)
```

Comment:
Yeah, the whole strong parameters thing is only useful if you're sending the whole hash to `.create()` or `#update()` or whatever. I think this reads simpler as

```ruby
email = params[:observation][:user][:email_address]
```